torch/_utils_internal.py

```diff
@@ -54,7 +54,7 @@ def get_source_lines_and_file(obj, error_msg=None):
                "available. Original error: {}".format(obj, e))
         if error_msg:
             msg += '\n' + error_msg
-        raise OSError(msg)
+        raise OSError(msg) from None
```

Comment:
Please consider using exception chaining and remove "Original error: {}" from the message (which seems to act as workaround for the lack of the feature in Python 2)

Reply:
Sure.